libyul/backends/evm/EVMDialect.cpp

```diff
@@ -222,3 +213,61 @@ SideEffects EVMDialect::sideEffectsOfInstruction(evmasm::Instruction _instructio
 		evmasm::SemanticInformation::invalidatesMemory(_instruction)
 	};
 }
+
+EVMDialectTyped::EVMDialectTyped(langutil::EVMVersion _evmVersion, bool _objectAccess):
```

Comment:
This dialect is currently used for "Yul" in the assembly stack. This means that `solc --yul` will use this.